recipes/ggd/post-link.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -eo pipefail -o nounset
+
+## Organize required conda channels
+conda config --add channels defaults
+conda config --add channels ggd-genomics
+conda config --add channels bioconda
+conda config --add channels conda-forge
+
+## initialize local metadata
+cat << EOF > initialize_metadata.py
```

Comment:
`conda` *does* execute the post-link scripts inside the installation prefix. But I'm not sure if we guarantee that.
To not rely on the current working directory, write to files inside the prefix, e.g., `>"${PREFIX}/unique-file-name`.
But in this case, you can avoid that temporary file altogether:
```sh
"${PREFIX}/bin/python" -c "$( cat <<'EOF'
# python code
EOF
)" >"${PREFIX}/.messages.txt" 2>&1
```
Also note the `"${PREFIX}/bin/python"` and `>"${PREFIX}/.messages.txt" 2>&1` parts: make sure to use the correct `python` binary and output any messages to `.messages.txt` or else `conda` won't display them to the user.

Then again, as noted above:
General advice: If anyhow possible, avoid post-link scripts.

Reply:
I have changed it per your suggestion. Rather than writing a new python script, executing it, and removing it, it now uses the -c command to do it from a string.